### ISSUE TITLE
Ошибка при выполнении слияния организаций

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2872]
Originally reported on Google Code with ID 2872

```
ОАО Лунинецлес
Нашли задвоенную организицию (ОАО "Пинское РСУ" и Пинское РСУ ОАО г.Пинск). Выполняем
слияние: ОАО "Пинское РСУ" оставляем, вторую удаляем.. 
В процессе слияния выдает ошибку: 
Невозможно произвести объединение. Процесс приостановлен.
Возникла следующая исключительная ситуация:
Исключение 34
The movement was made incorrect!



```

Reported by `spakovira.moved@tut.by` on 2012-08-28 06:27:55

**Comment by gsbelarus** [comment]
```

  http://code.google.com/p/gedemin/issues/detail?id=2692&can=1&q=%D1%81%D0%BB%D0%B8%D1%8F%D0%BD%D0%B8%D0%B5&colspec=ID%20Opened%20Closed%20Type%20Status%20Priority%20Reporter%20Owner%20Summary

```

Reported by `NikolayUkleyko` on 2012-08-28 11:18:39

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-09-14 13:30:49
- **Labels added**: Type-Other, Priority-Medium, Depot

**Comment by gsbelarus** [comment]
```
Давно хотелось сделать. В алгоритме слияния выставлять переменную контекста транзакции.
В заинтересованных триггерах проверять на эту переменную.
```

Reported by `gs1994` on 2012-09-19 12:59:49
- **Labels added**: GedeminEXE, DB

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-10-19 09:01:05
- **Status changed**: `Duplicate`
- **Merged into**: #2692
